src/idea.js

```diff
@@ -1,9 +1,9 @@
 class Idea {
-  constructor(title, body) {
+  constructor(title, body, star) {
```

Comment:
Still not sure if we need to pass this as a parameter, but am going to work a little more to see if I can't figure it out